components/ListView/ListItemImages.js

```diff
@@ -43,10 +46,43 @@ class ListItemImages extends React.Component {
     this.props.setModalDeleteImageVisible(true);
   }
 
+  handleLogsShow() {
+    this.setState(prevState => ({ logsExpanded: !prevState.logsExpanded, fetchingLogs: !prevState.logsExpanded }));
+    fetchComposeLogApi(this.props.listItem.id).then(
+      logs => {
+        this.setState({ logsContent: logs, fetchingLogs: false });
+      },
+      () => {
+        this.setState({
+          logsContent: <FormattedMessage defaultMessage="No anaconda log available" />,
+          fetchingLogs: false
+        });
+      }
+    );
+  }
+
   render() {
     const { listItem } = this.props;
     const timestamp = new Date(listItem.job_created * 1000);
     const formattedTime = timestamp.toDateString();
+    const moreButton = (
+      <a className="show-logs-button" role="presentation" onKeyDown={this.handleLogsShow} onClick={this.handleLogsShow}>
+        {this.state.logsExpanded && <FormattedMessage defaultMessage="Hide logs" />}
+        {!this.state.logsExpanded && <FormattedMessage defaultMessage="Show logs" />}
+      </a>
```

Comment:
I'm getting a `text` cursor instead of a `pointer` here.

Reply:
Try using `<button class="btn btn-link" ...>` instead of `<a className="show-logs-button" role="presentation" ... >` here. The `.btn.btn-link` element will have more padding around it, and you could remove the custom class and css. And it also has the correct cursor.

Reply:
Ah cool, thanks for the hint! Otherwise are you satisfied with the placement of thhis 'show more' button/link?

Do you think it's belongs next to the status?

Reply:
Plus, a `<button>` is semantically more correct than an `<a>` element in this case (e.g. if an `<a>` doesn't have an `href` value, then use a button instead)